### PR TITLE
my initial commit

### PR DESCRIPTION
**Pull request by shireeshatheegulla**

### CODE REVIEW

File: src/Assignment-5/assignment5.html [hunk 1]
```diff
@@ -12,7 +12,32 @@
...
+	</div>
+	<div class="card">
+	  <section>
+		<img src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/1145795/plant-4.png" alt="Some pointy plants">
+	  </section>
+	</div>
+	  <div class="card-content">
+		<h3>Some pointy ones</h3>
+		<p>Florem ipsum sugarbush bloom red rose waxflower coneflower ginger. Saxifrage forget-me-not obedient plant.</p>
+		<p><a href="#0">I'll take 10</a></p>
```
Comment (vivekkumar677) [inline src/Assignment-5/assignment5.html]: looks good but can remove extra <p> in line 35

File: src/Assignment-5/assignment5.html [hunk 2]
```diff
@@ -12,7 +12,32 @@
...
+		<!-- inspiration: https://dribbble.com/shots/3784003-Plant-description-page -->
+	<div class="card">
+	  <section>
+		<img src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/1145795/plant-1.png" alt="A leafy plant">
+	</section>
+	</div>
+	  <div class="card-content">
+		<h3>A leafy one</h3>
+		<p>Blossom dorset heath scabious ipsum. Grape hyacinth bee balm bird of paradise obedient plant african lily lily. Spring foxglove florist’s nighmare primrose.</p>
+		<p><a href="#0">So leafy</a></p>
```
Comment (vivekkumar677) [inline src/Assignment-5/assignment5.html]: here also can remove extra <p> content

File: src/Assignment-5/assignment5.css [hunk 3]
```diff
@@ -1,7 +1,70 @@
...
+  flex-wrap:wrap;
+}
+.card {
+  flex-direction: column;
+  margin-top: 50px;
+  margin-bottom: 50px;
+}
 
+section{
+  display:felx;
```
Comment (vivekkumar677) [inline src/Assignment-5/assignment5.css]: flex instead of felx

File: src/Assignment-5/assignment5.css [hunk 4]
```diff
@@ -1,7 +1,70 @@
-/* Our CSS file. */
+* {
+  box-sizing: border-box;
+}
+
+.container{
+  display:flex;
```
Comment (vivekkumar677) [inline src/Assignment-5/assignment5.css]: both css and html file looks like proper alignment. Good job,

File: src/Assignment-5/assignment5.html [hunk 5]
```diff
@@ -12,7 +12,32 @@
...
+	  </div>
+	</div>
+	<div class="card">
+	  <section>
+		<img src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/1145795/plant-4.png" alt="Some pointy plants">
+	  </section>
+	</div>
+	  <div class="card-content">
+		<h3>Some pointy ones</h3>
```
Comment (vivekkumar677) [inline src/Assignment-5/assignment5.html]: can use h1 tag instead of h3

**Review by vivekkumar677 — CHANGES_REQUESTED** [review]

**Comment by vivekkumar677** [comment]
Please make some changes and let me know i`ll approve

**Comment by Nocturnal22** [comment]
Dog static site : images not loading

**Pull request closed without merging**